runelite-client/src/main/java/net/runelite/client/plugins/grandexchange/GrandExchangeItems.java

```diff
@@ -35,13 +35,15 @@
 	private final int itemId;
 	private final int gePrice;
 	private final double haPrice;
+	private final int geItemLimit;
 
-	GrandExchangeItems(AsyncBufferedImage icon, String name, int itemId, int gePrice, double haPrice)
+	GrandExchangeItems(AsyncBufferedImage icon, String name, int itemId, int gePrice, double haPrice, int geItemLimit)
```

Comment:
Wait why is this constructor even here. Just remove it, the `@Data` annotation already creates constructor. and probably change `@Data` to `@Value` when at it